plugins/mcreator-localization/help/fr_FR/entity/trading_type.md

```diff
@@ -0,0 +1,5 @@
+La façon dont vous échangez avec l'entité.
+
+* **NONE:** Désactive les échanges.
+* **Villager:** L'entité a des niveaux (et donc, une barre de progression), comme les villageois vanilla. Par contre, il n'y a pas de bloc de profession.
+* **Wandering Trader:** L'entité n'a pas de niveau, elle échange comme le Marchand Ambulant.
```

Comment:
```suggestion
* **Wandering Trader:** L'entité n'a pas de niveau, elle échange comme le marchand ambulant.
```

Reply:
I checked other help tips, it always start with caps.